tests/jeos/firstrun.pm

```diff
@@ -328,23 +323,25 @@ sub run {
         assert_screen 're-encrypt-finished', 720 unless is_sle_micro('>=6.2');
     }
 
-    if (is_tumbleweed || is_microos || is_sle_micro('>6.0') || is_leap_micro('>6.0') || is_sle('>=16')) {
-        assert_screen 'jeos-ssh-enroll-or-not', 120;
-
-        if (get_var('SSH_ENROLL_PAIR')) {
-            mutex_wait 'dhcp';
-            sleep 30;    # make sure we have an IP
-            mutex_create 'SSH_ENROLL_PAIR';
-            send_key 'y';
-            check_screen 'jeos-ssh-enroll-pairing', 20;
-            assert_screen 'jeos-ssh-enroll-paired', 120;
-            send_key 'y';
-            assert_screen 'jeos-ssh-enroll-import', 120;
-            send_key 'y';
-            assert_screen 'jeos-ssh-enroll-imported', 120;
-            send_key 'ret';
-        } else {
-            send_key 'n';
+    if (get_var('WSL_VERSION') || is_tumbleweed || is_microos || is_sle_micro('>6.0') || is_leap_micro('>6.0') || is_sle('>=16')) {
+        if (!get_var('WSL_VERSION')) {
```

Comment:
We have the main `if` that checks if WSL_VERSION is present and then a big inner `if` when WSL is not present.

Are you sure we need the main `if`?

Reply:
ok.